samples/bluetooth/peripheral_hids_keyboard/README.rst

```diff
@@ -251,13 +251,14 @@ To test with the `Bluetooth Low Energy app`_, complete the following steps:
          The first one has the value ``02000F0000000000``, the second has the value ``0200000000000000``.
          These values correspond to press and release of character "l" with the Shift key pressed.
       #. In the app, select the HID Report (which has UUID ``0x2A4D`` and the properties ``Read``, ``WriteWithoutResponse``, and ``Write``).
-         Enter ``02`` in the text box and click the :guilabel:`tick mark` button.
-         This sets the modifier bit of the Output Report to 02, which simulates turning Caps Lock ON.
+         Enter any number that has bit 1 set to ``1`` (e.g. ``02``) in the text box and click the :guilabel:`tick mark` button.
```

Comment:
Consider turning on/off only Caps Lock LED here (as other output report bits are not used by the sample):
```suggestion
         Enter ``02`` in the text box and click the :guilabel:`tick mark` button.
```
Also in other places

Reply:
I actually mention that other bits are unused. I think the current version gives the user better understanding of the topic and we still suggest to enter 02. The whole issue comes from the fact that it was not clear for users what other values do and why,